src/intl/en/page-staking.json

```
@@ -187,7 +187,7 @@
   "page-staking-section-comparison-requirements-title": "Requirements",
   "page-staking-section-comparison-solo-requirements-li1": "You must deposit 32 ETH",
   "page-staking-section-comparison-solo-requirements-li2": "Maintain hardware that runs both an Ethereum execution client and consensus client while connected to the internet",
-  "page-staking-section-comparison-solo-requirements-li3": "The <a href=\"https://goerli.launchpad.ethereum.org\" target=\"_blank\">Staking Launchpad</a> will walk you through the process and hardware requirements",
+  "page-staking-section-comparison-solo-requirements-li3": "The <a href=\"https://launchpad.ethereum.org\" target=\"_blank\">Staking Launchpad</a> will walk you through the process and hardware requirements",
```

```suggestion
  "page-staking-section-comparison-solo-requirements-li3": "The <a href=\"https://holesky.launchpad.ethereum.org\" target=\"_blank\">Staking Launchpad</a> will walk you through the process and hardware requirements",
```
Thanks @zarifpour 

I think this was supposed to go to a testnet launchpad. Agree with the change, but I think we should go to [https://holesky.launchpad.ethereum.org/en/](https://holesky.launchpad.ethereum.org/en/).

@wackerow do you have any opinion here?

---

Holešky please =)

This was part of an effort over a year ago to direct people to the testnet version of the launchpad by default.